acoustid/data/fingerprint.py

```
@@ -2,10 +2,11 @@
 # Distributed under the MIT license, see the LICENSE file for details.
 
 import logging
-from typing import Any, Dict, List, NamedTuple, Optional
+from typing import Any, Dict, List, NamedTuple, Optional, cast
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)